Sources_v3/StreamChatUI/MessageActionsPopup/MessageActionsView+ActionButton.swift

```diff
@@ -24,16 +25,28 @@ extension MessageActionsView {
             
             addTarget(self, action: #selector(touchUpInsideHandler(_:)), for: .touchUpInside)
         }
+
+        override open func tintColorDidChange() {
+            super.tintColorDidChange()
+            
+            updateContent()
```

Comment:
`updateContentIfNeeded` 😱

Reply:
We should make a SwiftLint for this! -> https://github.com/realm/SwiftLint#defining-custom-rules

Reply:
@VojtaStavik @ManWithBear that's a good question 😄 

I don't think we should invoke `updateContentIfNeeded` from `tintColorDidChange` since not all views/controls change/use `tintColor` when content is changed.

Feels like some kind of `updateAppearance` API asks to be introduced where we can do all styling.

Reply:
> We should make a SwiftLint for this! -> https://github.com/realm/SwiftLint#defining-custom-rules

https://stream-io.atlassian.net/browse/CIS-535